paasta_tools/cli/cmds/status.py

```diff
@@ -534,6 +573,13 @@ def report_status_for_cluster(
                 for line in status.rstrip().split('\n'):
                     output.append('    %s' % line)
 
+    for name, instance in instance_whitelist.items():
+        if str(instance) == "<class 'paasta_tools.tron_tools.TronActionConfig'>":
```

Comment:
`isinstance(instance, tron_tools.TronActionConfig)`

Reply:
I tried but this doesn't work unfortunately.